### ISSUE TITLE
Set up a way to make donations

### ISSUE THREAD

**Issue by paradox460** [t9md/atom-vim-mode-plus#877]
Vim Mode Plus is an incredible plugin, and has saved me countless hours of developer time.

As such, I would like to give back a bit to the project, at the very least buy you a beer.

Could you set up a donation link?

**Comment by bronson** [comment]
I agree.  Check out the green badge at the top of vmp's readme: https://www.bountysource.com/teams/atom-vim-mode-plus

If you can think of a way of making it more obvious without making it more in-your-face, that could be worth a PR.

**Comment by t9md** [comment]
@paradox460 
Thanks for kind comment. I welcome donation at any time!

@bronson 
Thank you! I haven't said you explicitly, I know you've made big amount of donation in total from very early stage of vmp.
That's really motivate me, thanks thanks thanks!

**Comment by t9md** [comment]
@paradox460 Thanks for your bounty, will keep working to make vmp better!.
